src/popup/infoPopupModule.js

```diff
@@ -127,26 +130,31 @@ function getPinterestShareLink(sourceLink, imageLink) {
   return `https://pinterest.com/pin/create/button/?url=${sourceLink}&media=${imageLink}&description=I%20found%20an%20image%20through%20CC%20search%20%40creativecommons%3A%20${sourceLink}`;
 }
 
+// Added the share link for tumblr
+function getTumblrShareLink(sourceLink,imageLink) {
```

Comment:
Require space after the `,`. [comma-spacing](https://eslint.org/docs/rules/comma-spacing)